week-2-practise/level-3/6-map.js

```diff
@@ -9,13 +9,14 @@ function tidyUpString(str) {
     .toLowerCase()
     .replace("/", "");
 }
-
+// var mentorsTidy = mentors.map(tidyUpString) is there is a difference between this var to be here or to be as it is now betwen the two vars
 function captialise(str) {
-  // complete this function
+  return str.charAt(0).toUpperCase() + str.slice(1); // complete this function
 }
 
 var mentors = ["/Daniel ", "irina ", " Gordon", "ashleigh "];
-var mentorsTidyAndCapitalised;
+var mentorsTidy = mentors.map(tidyUpString);
+var mentorsTidyAndCapitalised = mentorsTidy.map(captialise);
 
 console.log(mentorsTidyAndCapitalised);
```

Comment:
excellent but be careful not to introduce "typos" like here. Capitalise is spelled incorrectly. It is always worth correcting this type of error.

Reply:
I think this is actually a mistake in the example code. It's since been fixed.

Reply:
yes  you are right this time isn't my mistake :D

Reply:
thanks Luke - yep, please ignore Youssef!